server/events/instrumented_project_command_runner.go

```diff
@@ -35,30 +35,31 @@ func NewInstrumentedProjectCommandRunner(scope tally.Scope, projectCommandRunner
 }
 
 func (p *InstrumentedProjectCommandRunner) Plan(ctx command.ProjectContext) command.ProjectResult {
-	return RunAndEmitStats("plan", ctx, p.projectCommandRunner.Plan, p.scope)
+	return RunAndEmitStats(ctx, p.projectCommandRunner.Plan, p.scope)
```

Comment:
I didn't see this comment, let me check

Reply:
I don't know how to develop the test 🥺

Reply:
For now @albertorm95 , please look to other tests for examples.

https://github.com/runatlantis/atlantis/blob/23443a9262c3817d01f397636d4387ba4e392b41/server/events/import_command_runner_test.go

Then we can create `server/events/instrumented_project_command_runner_test.go` and then write a test that will test the output of the `RunAndEmitStats` function.

Reply:
Working on it

Reply:
@nitrocode I try to develop the code, but it looks like there is a lot of work to do in order to make that `NewInstrumentedProjectCommandRunner` or `RunAndEmitStats` testable, looks like I need to develop a mock? and then add it to the `events.NewPlanCommandRunner()` and same for apply, etc.., I don't feel confident enough to do this

The latest release is already failing because of this. What do you suggest to do in this case?

Reply:
I'm tempted to merge based on the testing.

cc: @GenPage @jamengual thoughts?